dev-tools/dependencies-report

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -e
+set -x
+
+SRCPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+outfile=dependencies.csv
+
+while :; do
+	case $1 in
+		--csv)
+			if [ "$2" ]; then
+				outfile=$2
+			else
+				echo "ERROR: --csv needs a non-empty argument"
+				exit 1
+			fi
+			shift
+			;;
+		--csv=?*)
+			outfile=${1#*=}
+			;;
+		--csv=)
+			echo "ERROR: --csv needs a non-empty argument"
+			exit 1
+			;;
+		*)
+			break
+			;;
+	esac
+
+	shift
+done
+
+go mod tidy
```

Comment:
There is a `mage notice` target that does this already.

Reply:
thing is the output file is different and notice does not allow it to be overriden.